.eslintignore

```diff
@@ -0,0 +1 @@
+!.eslintrc.cjs
```

Comment:
I prefer enable for all dot files

Reply:
Updated in https://github.com/chatbotgang/image-area-selector/pull/1/commits/4eeff0de03427758dc53bfcadcb297c3f3c38b10#diff-a0fdacd2ade87c5238dde44378f574f7e123e669acdf8b740878b14b33b20275R1